pages/linux/hcitool.md

```diff
@@ -0,0 +1,36 @@
+# hcitool
+
+> Monitor & Configure Bluetooth connections.
+> Send special commands to Bluetooth devices.
```

Comment:
```suggestion
> Monitor, configure connections and send special commands to Bluetooth devices.
> More information: <https://manned.org/hcitool>.
```

Reply:
Thanks for the suggestions @CleanMachine1